### PR TITLE
building module files structure

### PR DESCRIPTION
**Pull request by abdulwahidsharief**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+  "name": "drupal/drupal_commerce_razorpay",
+  "type": "drupal-module",
+  "description": "Provides Commerce integration for Razorpay.",
+  "keywords": ["Drupal"],
+  "minimum-stability": "dev",
+  "homepage": "http://drupal.org/project/drupal_commerce_razorpay",
```
Comment (sofiya02) [inline composer.json:7]: have you check the availability of homepage URL?

File: src/PluginForm/RazorpayForm.php [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+class RazorpayForm extends BasePaymentOffsiteForm
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function buildConfigurationForm(array $form, FormStateInterface $form_state)
+    {
+        $form = parent::buildConfigurationForm($form, $form_state);
+
+        return $this->buildRedirectForm($form, $form_state,'',[],self::REDIRECT_POST);
```
Comment (sofiya02) [inline src/PluginForm/RazorpayForm.php:23]: no space found after comma in argument list

File: src/Plugin/Commerce/PaymentGateway/RazorpayCheckout.php [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+use Drupal\commerce_order\Entity\OrderInterface;
+use Drupal\commerce_payment\Entity\PaymentInterface;
+use Drupal\commerce_payment\Entity\PaymentMethodInterface;
+use Drupal\commerce_payment\Exception\HardDeclineException;
+use Drupal\commerce_payment\Exception\PaymentGatewayException;
+use Drupal\commerce_payment\PaymentMethodStorageInterface;
+use Drupal\commerce_payment\Plugin\Commerce\PaymentGateway\OffsitePaymentGatewayBase;
+use Drupal\commerce_price\Price;
+use Drupal\Component\Serialization\Json;
+use Drupal\Core\Form\FormStateInterface;
```
Comment (sofiya02) [inline src/Plugin/Commerce/PaymentGateway/RazorpayCheckout.php:14]: Remove unwanted lib

File: src/Controller/RzpController.php [hunk 4]
```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace Drupal\drupal_commerce_razorpay\Controller;
+
+use Drupal\commerce_order\Entity\Order;
+use Drupal\Core\Controller\ControllerBase;
+use Drupal\Core\Url;
+use Razorpay\Api\Api;
```
Comment (sofiya02) [inline src/Controller/RzpController.php:8]: remove unwanted lib

**Review by sofiya02 — CHANGES_REQUESTED** [review]

File: composer.json (hunk 1 above)
Comment (abdulwahidsharief) [reply composer.json:7]: yes shyamala has checked and suggested the repo name

**Review by sofiya02 — APPROVED** [review]
LGTM

**Pull request merged**